Pipfile

```diff
@@ -22,7 +22,7 @@ flask-script = "~=2.0.6"
 flask-cors = "~=3.0.9"
 confluent-kafka = "~=1.5.0"
 requests = "~=2.25.1"
-insights-core = {git = "https://github.com/RedHatInsights/insights-core.git",ref = "3.0"}
+insights-core = "==3.0.265"
```

Comment:
```suggestion
insights-core = "~=3.0.265"
```

Reply:
@apuntamb Pushed the suggested chance, along with updated lock file :+1: